### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by AndreaM2429**
Hello there!!
I'm sending this PR to you for your review, feedback, and approval. I'm implementing new features that would like to merge into the `develop` branch.

# 🎯 Milestones:

⚜️ Create a new rails app.
⚜️ Install `devise` to manage authentication.
⚜️ Install `bullet` to prevent N+1 problems.
⚜️ Install `letter_opener` to manage the confirmation emails.
⚜️ Install RoR linters.
⚜️ Add the basic and general project information.
⚜️ Create the migration files
⚜️ Create models, controllers, the files corresponding with the index views, and routes for the app.

## 🏁 General requirements 🏁:

❇️ Gitflow
❇️ No linter errors
❇️ Documented work
❇️ Follow best practices

☆.｡.:*･ﾟ☆.｡.:*･ﾟ☆.｡.:*･ﾟ☆.｡.:*･ﾟ☆.｡.:*･ﾟ☆.｡.:*･ﾟ☆.｡.:*･ﾟ☆☆.｡.:*･ﾟ☆.｡.:*･ﾟ☆.｡.:*･ﾟ☆.｡.:*･ﾟ☆.｡.:*･ﾟ☆.｡.:*･ﾟ☆.｡.:*･ﾟ☆☆.｡.:*･ﾟ☆.

🙇‍♀️🙇‍♀️Thanks in advance for taking the time and all your effort to review these new implementations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+  <ul>
+    <li><a href="https://github.com/microverseinc/linters-config/tree/master/ror">RoR linters</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **Database based on a this [ERD](app/assets/images/budgetApp.drawio.png)**
+
```
Comment (NitBravoA92) [inline README.md]: [OPTIONAL] In order to have clean and error-free documentation, I kindly suggest you remove the letter "a" from this sentence. Example:

- **Database based on this [ERD](app/assets/images/budgetApp.drawio.png)**

File: README.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+## 💻 Getting Started <a name="getting-started"></a>
+
+This repository includes files with Ruby lenguage.
+
+### Prerequisites
+
+In order to run this project you need to have install:
+- Ruby check [this](https://www.ruby-lang.org/en/) to install it.
+- Rails check [this](https://www.postgresql.org/) to install it.
+- Postgresql check [this](https://guides.rubyonrails.org/) to install it.
```
Comment (NitBravoA92) [inline README.md]: [OPTIONAL] In order to have the documentation with links that lead to the correct pages,  I kindly suggest you check if the PostgreSQL and Rails links are defined correctly.

**Review by NitBravoA92 — APPROVED** [review]
# Approved :green_circle: 
Hi @AndreaM2429 

## Congratulations! 🎉 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 



## Highlights :heavy_check_mark: 
- [x] Rails project has a correct configuration :white_check_mark:  
- [x] All models created with their validations :white_check_mark: 
- [x] Routes created successfully :white_check_mark: 
- [x] Controllers and Views :white_check_mark: 
- [x] No linters errors :white_check_mark:  
- [x] Pull request has a descriptive title and summary :white_check_mark: 
- [x] Repository has a professional readme file :white_check_mark:  

## Optional suggestions
- Please read the comments I have left below.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------

File: README.md (hunk 1 above)
Comment (AndreaM2429) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (AndreaM2429) [reply README.md]: Thanks for the kind observation!

**Comment by AndreaM2429** [comment]
Thanks, @NitBravoA92 for your suggestions and time to review my project.

**Comment by AndreaM2429** [comment]
Thanks, @NitBravoA92 for your suggestions and time to review my project.

**Pull request merged**